apps-rendering/src/components/Callout/formFields.tsx

```diff
@@ -0,0 +1,148 @@
+import type { FormField as formFieldType } from '@guardian/apps-rendering-api-models/formField';
+import {
+	Option,
+	Select,
+	TextArea,
+	TextInput,
+} from '@guardian/source-react-components';
+import { FileInput } from '@guardian/source-react-components-development-kitchen';
+import CheckboxInput from 'components/CheckboxInput';
+import RadioInput from 'components/RadioInput';
+import type { FC } from 'react';
+import { logger } from '../../logger';
+import type { FormDataType, ValidationErrors } from './calloutForm';
+import { fieldInput, textarea } from './styles';
+
+type FormFieldProp = {
+	formId: number;
+	formField: formFieldType;
+	formData: FormDataType;
+	setFieldInFormData: (
+		id: string,
+		data: string | string[] | undefined,
+	) => void;
+	validationErrors: ValidationErrors;
+};
+
+export const FormField: FC<FormFieldProp> = ({
+	formId,
+	formField,
+	formData,
+	setFieldInFormData,
+	validationErrors,
+}) => {
+	const { type, label, description, mandatory, options, id } = formField;
```

Comment:
do we want to be using the `hideLabel` property from the forms? I can't see why it would be used but if it's there, we may as well pass it in?

Reply:
Mind you, do you even get the `hideLabel` property from the model in AR?

Reply:
We do not get the property from AR